packages/screens/LaunchpadERC20/LaunchpadERC20Sales/LaunchpadERC20CreateSaleForm.tsx

```diff
@@ -0,0 +1,200 @@
+      <BrandText style={fontSemibold20}>Sale Informations</BrandText>
+
+      <SpacerColumn size={1} />
+
+      <BrandText style={[fontSemibold14, { color: neutral77 }]}>
+        Information about your Sale
+      </BrandText>
+
+      <SpacerColumn size={2.5} />
+
+      <TextInputCustom<TCreateSaleForm>
+        label="Token Name (The token have to be owned by the caller)"
+        name="tokenName"
+        fullWidth
+        placeholder="Type the token name here..."
+        variant="labelOutside"
+        control={control}
+        rules={{ required: true }}
+      />
+
+      <SpacerColumn size={2.5} />
+
+      <TextInputCustom<TCreateSaleForm>
+        label="Price per unit of token ($GNOT)"
+        name="pricePerToken"
+        fullWidth
+        placeholder="Type the price per token here..."
+        variant="labelOutside"
+        control={control}
+        rules={{ required: true }}
+      />
+
+      <SpacerColumn size={2.5} />
+
+      <TextInputCustom<TCreateSaleForm>
+        label="Limit of tokens a user can buy"
+        name="limitPerAddr"
+        fullWidth
+        placeholder="Type the limit of tokens a user can buy here..."
+        variant="labelOutside"
+        control={control}
+        rules={{ required: true }}
+      />
+
+      <SpacerColumn size={2.5} />
+
+      <TextInputCustom<TCreateSaleForm>
+        label="Minimum Goal (The sale will refund buyers if this goal is reach)"
+        name="minGoal"
+        fullWidth
+        placeholder="Type the minimum goal here..."
+        variant="labelOutside"
+        control={control}
+        rules={{ required: true }}
+      />
+
+      <SpacerColumn size={2.5} />
+
+      <TextInputCustom
+        label="Maximum Goal (The sale will stop if this goal is reach)"
+        name="maxGoal"
+        fullWidth
+        placeholder="Type the maximum goal here..."
+        variant="labelOutside"
+        control={control}
+        rules={{ required: true }}
+      />
+
+      <SpacerColumn size={2.5} />
+
+      <Controller<TCreateSaleForm>
+        name="startTimestamp"
+        control={control}
+        render={({ field: { onChange } }) => (
+          <DateTimeInput
+            label="Start date"
+            onChange={onChange}
+            timestamp={startTimestamp}
+            isDirty={getFieldState("startTimestamp").isDirty}
+          />
+        )}
+      />
+
+      <SpacerColumn size={2.5} />
+
+      <Controller<TCreateSaleForm>
+        name="endTimestamp"
+        control={control}
+        render={({ field: { onChange } }) => (
+          <DateTimeInput
+            label="End date"
+            onChange={onChange}
+            timestamp={endTimestamp}
+            isDirty={getFieldState("endTimestamp").isDirty}
+          />
+        )}
+      />
+      <SpacerColumn size={2.5} />
+
+      <BrandText style={[fontSemibold14, { color: neutral77 }]}>
+        Whitelisted addresses (Optional)
+      </BrandText>
```

Comment:
We have a `Label` component.
I also suggest to add a sublabel to explain what is this input
Don't forget to import
```suggestion
      <Label isRequired>
        Whitelisted addresses
      </Label>
      <SpacerColumn size={1} />
      <BrandText style={[fontSemibold14, { color: neutral77 }]}>
        Select a TXT or CSV file that contains the whitelisted addresses (One
        address per line)
      </BrandText>
```

Reply:
[8a58d4d](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/8a58d4dda4cf108aac3e512c9716bb0e926e37aa)